packages/fxa-profile-server/lib/server/web.js

```diff
@@ -4,7 +4,7 @@
 
 const Hapi = require('@hapi/hapi');
 const Raven = require('raven');
-const cloneDeep = require('lodash.clonedeep');
```

Comment:
this dep is 4 years old and seems like it was not updated in Sync with the latest lodash (and is causing problems with latest es6 changes in joi)